client/src/components/schedule_table/ScheduleTable.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent, useState, useMemo, useContext } from 'react';
+import { withTheme } from '@material-ui/core/styles';
+import styled from 'styled-components';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+
+interface LeagueScheduleProps {
+  className?: string;
+}
+const UnstyledLeagueSchedule: FunctionComponent<LeagueScheduleProps> = ({
```

Comment:
what is this file??

Reply:
oh,,, I forgot to delete the file,,,, Thank you for letting me know!